app/scripts/modules/core/src/account/AccountSelectInput.tsx

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 import { $q } from 'ngimport';
 import { flatten, isEqual, map, uniq, xor } from 'lodash';
-import Select, { Option } from 'react-select';
+import { Option } from 'react-select';
 
 import { createFakeReactSyntheticEvent } from 'core/presentation/forms/inputs/utils';
-import { IFormInputProps } from '../presentation';
+import { IFormInputProps } from 'core/presentation/forms/inputs';
+import { ReactSelectInput } from 'core/presentation/forms/inputs/ReactSelectInput';
```

Comment:
I had to import deeply to avoid a circular module dependency 😢 
Some day we'll build these packages with rollup and find all these circular dependencies (there will be a LOT)